subworkflows/local/analyse_transcripts/main.nf

```diff
@@ -29,38 +31,51 @@ workflow ANALYSE_TRANSCRIPTS {
     case_info                     //   channel: [optional]  [ val(case_id) ]
     skip_drop_ae                  // parameter: [mandatory] default: false
     skip_drop_as                  // parameter: [mandatory] default: false
+    skip_drop_mae                 // parameter: [mandatory] calculated from samplesheet
     skip_export_counts_drop       // parameter: [mandatory] default: true
+    ch_drop_mae_high_q_vcf_tbi        //   channel: [ path(ch_high_q_vcf), path(ch_high_q_vcf_tbi) ]
     skip_stringtie                // parameter: [mandatory] default: false
 
     main:
     ch_versions = Channel.empty()
 
     ch_bam_ds_bai
```

Comment:
Took me a second to figure out what the `_ds_` part stood for 😆 . Maybe a bit odd to use in the middle of the channel name.  Would favour a longer but more descriptive channel name such as `ch_downsampled_bam_bai` or even remove the `_ds_` part all together in this subworkflow. Anyway, nothing we need to adress or discuss in this PR 😸

Reply:
Sounds good, lets take that one later